docs: how to become tsc member (#449)
asyncapi/community: TSC_MEMBERSHIP.md

## Why the TSC was formed
The concept of the TSC was formed when AsyncAPI joined the Linux Foundation in 2021 to guarantee a system of open governance that promotes ownership and neutrality of all participants in the initiative.
 
Some of the principles of the Linux Foundation partnership include:

* No founding members with special privileges
* Equality among all contributors
* Asynchronous decision making 
* Every maintainer is allowed to automatically become a TSC member

## Benefits of becoming a TSC member

As a TSC member you will enjoy the following benefits: 
 
* Recognition from AsyncAPI community
* Exposure of your portfolio to potential employers

* Involved in the voting process on topics concerning the entire Initiative
* Included in key decision-making for the Initiative. Especially related to finance. 

## How to become a TSC member
Any maintainer can become a TSC member. In other words, to become a TSC member, you need to become a maintainer of one of the projects in the AsyncAPI Initiative. 
You can become a maintainer by doing at least one of the following items within the AsyncAPI Initiative GitHub organization:

> * Initiate a new repository/project and become its maintainer

Reviewer: for now document `Become a maintainer in an existing project` also covers `Initiate a new repository/project and become its maintainer`, so link it here too please

 
Some of the principles of the Linux Foundation partnership include:

* No founding members with special privileges
* Equality among all contributors
* Asynchronous decision making 
* Every maintainer is allowed to automatically become a TSC member

## Benefits of becoming a TSC member

As a TSC member you will enjoy the following benefits: 
 
* Recognition from AsyncAPI community
* Exposure of your portfolio to potential employers

* Involved in the voting process on topics concerning the entire Initiative
* Included in key decision-making for the Initiative. Especially related to finance. 

## How to become a TSC member
Any maintainer can become a TSC member. In other words, to become a TSC member, you need to become a maintainer of one of the projects in the AsyncAPI Initiative. 
You can become a maintainer by doing at least one of the following items within the AsyncAPI Initiative GitHub organization:
* Initiate a new repository/project and become its maintainer
* Donate existing repository/project and stay as its maintainer
* Support and commit to existing projects and be invited by other maintainers to join them as a maintainer

> ## Make your first contribution

Author: Hi @derberg.Great suggestions. Yes, I need your input for **Become a maintainer by donating a project**. Thanks

> ## Make your first contribution

Reviewer: @Dindihub the best will be to just link to document that will be produced by https://github.com/asyncapi/community/pull/442/files PR

in other words, I suggest `Make your first contribution` aka `Become maintainer in existing project` -> just extract it to a separate markdown document in the same PR

then, in the section where you list different ways to become TSC, just link to `Become maintainer in existing project` or link to the document produced by the other PR (there will be one document that covers new and donated project.

I hope that makes sense

> ## Make your first contribution

Author: Hi @derberg. 

If  I understand you correctly, you would like the whole section  **Make your first contribution** to be renamed to **Become Maintainer in Existing Project** and then extracted to a new file in the folder on PR #442 

Then link the document to **How to become a TSC member section**

However, I don't know how to create a file on someone else's PR. Any ideas?

> ## Make your first contribution

Reviewer: > extracted to a new file in the folder on PR https://github.com/asyncapi/community/pull/442

extracted but under your PR

> ## Make your first contribution

Author: @derberg A new PR or #449 ?

> ## Make your first contribution

Reviewer: this one, sorry for not being clear 😅

> ## Make your first contribution

Author: Hey @derberg . I have made the changes, please review. Let me know if that's how you want it. Thanks

* Ask maintainers to do a public live stream called **Let's talk about contributing** where they can explain a specific part of the project where you want to contribute.

**Step 2: Open a pull request**

To open a pull request successfully, follow our [Git workflows contributors guide](https://github.com/asyncapi/community/blob/master/git-workflow.md).

Use the following steps to open your pull request:
1. Configure a fork
2. Work on it locally 
3. Write an appropriate PR title using the [conventional commits specification](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md#conventional-commits)
4.Test your PR against Github checks to ensure they all pass

**Step 3: Get pull request merged**

After making a pull request, allow the maintainers to review your PR. If no one is alerted to review your PR, feel free to bring the issue to the maintainer's attention by pinging them on Slack or GitHub. We sometimes delay in response for many reasons; no one intends to ignore you 🙏🏼 .

Make sure all your PR checks pass (turn to green) to ensure your PR is merged quickly.   

**Step 4: Invitation to become a maintainer**

> After making several contributions to AsyncAPI, maintainers will recognize you. You will then receive an invitation to join the TSC. Look at this [maintainer invitation example from the website repository](https://github.com/asyncapi/website/pull/890).

Reviewer: ```suggestion
After making several contributions to AsyncAPI, maintainers will recognize you. You will then receive an invitation to join the TSC. Look at this [maintainer invitation example from the website repository](https://github.com/asyncapi/website/pull/890). If you feel that you contributed enough to become a maintainer, but maintainers did not recognize it, please open a GitHub issue in a related repository and indicate what you contributed so far and ask what else is needed to become a maintainer.
```

# BECOME A  TSC MEMBER!

In this article you will learn how to become a member of AsyncAPI Technical Steering Committee (TSC).You will know how to contribute, how to become a committer(maintainer), and realise the benefits of being a TSC member.

The following areas are covered:
* What is the TSC?
* Why TSC was formed
* Benefits of becoming a TSC member
* How to become a TSC member

If you prefer video, watch our [Youtube video](https://www.youtube.com/watch?v=uG_aLF9Z1F0) on how to become a TSC member.

> Definition of common terms in this document:

Reviewer: this is the most critical part of entire document. This vocabulary is tricky

I suggest everywhere in the document we talk about "maintainers" as this is what people commonly understand, that you have contributors and maintainers. This document is like a "nicer, more clear" version of https://github.com/asyncapi/community/blob/master/CHARTER.md that is written in a very official way. 

So I suggest just have 2 words here, Maintainer and Contributor, and that Maintainer for us is the same as what is described as Committer in https://github.com/asyncapi/community/blob/master/CHARTER.md and the same what in GitHub is called CODEOWNER (and link to https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)

Definition of common terms in this document:

> * **Maintainers:** Are Contributors who have earned the ability to modify (“commit”) source code, documentation, or other technical artifacts in a project’s repository (“Committers”); They are aslo reffered to as CODEOWNERS on [GitHub]()

Reviewer: you forgot to place a link to github docs about codeowners 😉

Author: Ah! Thanks

* **Contributers:** Include anyone in the community that contributes code, documentation, pull request reviews, or other technical artifacts to the Project (“Contributors”);

## What is the Technical Steering Committee?

> TSC is a group of maintainers and Committers that contribute to the creation and maintenance of AsyncAPI initiative. 

Reviewer: ```suggestion
TSC is a group of maintainers and Committers that contribute to the creation and maintenance of AsyncAPI Initiative.
```

## Why the TSC was formed

> AsyncAPI joined the Linux foundation in 2021 to guarantee a system of open governance that promotes ownership and neutrality of all participants in the initiative. 

Reviewer: ```suggestion
AsyncAPI joined the Linux Foundation in 2021 to guarantee a system of open governance that promotes ownership and neutrality of all participants in the initiative. 

```

> Some of the principles of the Linux foundation partnership include:

Reviewer: ```suggestion
Some of the principles of the Linux Foundation partnership include:

```

* Equality among all contributors
* Asynchronous decision making 
* Every committer is a TSC member

## Benefits of becoming a TSC member

Here are the benefits to becoming a  TSC member: 
 
* Recognition from AsyncAPI community
* Exposure for your portfolio to employers and company

> * Involved in voting process

Reviewer: ```suggestion
* Involved in the voting process on topics concerning entire Initiative

```

> * Included in key decision making

Reviewer: ```suggestion
* Included in key decision-making for the Initiative. Especially related to finance. 
```

## How to become a TSC member

> Anyone can be a TSC member. You however have to fulfill the following requirements:

Reviewer: ```suggestion
Any maintainer can be a TSC member. In other words, to become a TSC member, you need to become a maintainer in one of the projects in the AsyncAPI Initiative. 

You can do it by doing at least one of below things in AsyncAPI Initiative GitHub organization:
```

> * Involved in the voting process on topics concerning entire Initiative
> 
> * Included in key decision-making for the Initiative. Especially related to finance. 

Reviewer: ```suggestion
* Involved in the voting process on topics concerning the entire Initiative
* Included in key decision-making for the Initiative. Especially related to finance. 

```

> * Initiate a repository 

Reviewer: ```suggestion
* Initiate a new repository/project and become its maintainer
```

> * Donate projects to the initiative

Reviewer: ```suggestion
* Donate existing repository/project and stay as its maintainer
```

> * Support and commit to existing projects

Reviewer: ```suggestion
* Support and commit to existing projects and be invited by other maintainers to join them as a maintainer

```

> You can open issues on `bugs`,`docs`,`ideas`,and `reviews`. Pick an issue you would like to work on and follow the steps below to make your first contribution. 

Reviewer: this sentence doesn't fit this section somehow

Author: ok. I'll delete it.

## Make your first contribution
We know you are nervous to make your first contribution. In this section we guide you on steps to becoming a TSC member:

1. Pick an issue
2. Open a pull request
3. Get it merged
4. Get invitation to be a code owner
5. Become a TSC member

**Step 1: Pick an Issue**

Here's how to select an issue to contribute to:
* Join existing PR reviews

> * Look for issues with **good first issue label**

Reviewer: ```suggestion
* Look for issues with [**good first issue** label](https://github.com/issues?page=1&q=is%3Aopen+org%3Aasyncapi+sort%3Aupdated-desc+label%3A%22good+first+issue%22)
```

> * Join AsyncAPI slack community and join the `#11_how-to-contribute` channel

Reviewer: ```suggestion
* Join [AsyncAPI Slack community](https://asyncapi.com/slack-invite) and join the `#11_how-to-contribute` channel
```

> * Join a contributor e.g to do a live stream

Reviewer: I think we can remove this one

Reviewer: or modify it like 
```suggestion
* Ask maintainers to do a public live stream called **Let's talk about contributing** where then can explain specific part of the project where you want to contribute
```

**Step 2: Open a pull request**

To open a pull request successfully, follow our [Git workflows](https://github.com/asyncapi/community/blob/master/git-workflow.md) contributors guide.

Use the following steps to open your pull request:
1. Configure a fork
2. Work on it locally 

> 3. Write an appropriate PR title 

Reviewer: ```suggestion
3. Write an appropriate PR title using [conventional commits specification](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md#conventional-commits)

```

4. Test PR against Github checks-ensure they all pass

**Step 3: Get it merged**

> After making a pull request,allow the maintainers/code owners to review your PR.Incase no one is alerted to review your PR,feel free to bring the issue to the maintainers attention by either pinging them on slack or Github. 

Reviewer: ```suggestion
After making a pull request, allow the maintainers to review your PR. In case no one is alerted to review your PR, feel free to bring the issue to the maintainer's attention by either pinging them on Slack or GitHub. We are sometimes delayed in response for many different reasons, no one intention is to ignore you 🙏🏼 .

```

Make sure all your PR checks pass(turn to green) to ensure your PR is merged quickly.   

> **Step 4: Invitation to become a code owner**

Reviewer: ```suggestion
**Step 4: Invitation to become a maintainer**

```

> After making several contributions to AsyncAPI,the code owners will recognize you. You will then receive an invitation to join the TSC.  

Reviewer: ```suggestion
After making several contributions to AsyncAPI, maintainers will recognize you. You will then receive an invitation to join the TSC. Look at [this example from the website repository](https://github.com/asyncapi/website/pull/890).
```

> 3. Write an appropriate PR title using [conventional commits specification](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md#conventional-commits)
> 
> 4. Test PR against Github checks-ensure they all pass

Reviewer: ```suggestion
3. Write an appropriate PR title using [conventional commits specification](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md#conventional-commits)
4. Test PR against Github checks-ensure they all pass
```

**Step 5: Become a TSC member**

> Become a TSC member and join other 24 members. You will enjoy the following privileges:

Reviewer: we have this content already in one of first sections, my proposal for step 5 is

```
Once you become a maintainer in one of the repositories, you can apply to join a group of [TSC members](https://www.asyncapi.com/community/tsc).

Open a PR in **community** repository to add your name to [the list of members](https://github.com/asyncapi/community/blob/master/TSC_MEMBERS.json). Specify your name, repositories that you maintain and if you are sponsored by some company. For example [see this PR that introduced one of new members](https://github.com/asyncapi/community/pull/277).

Joining TSC is something that you are allowed to do as a maintainer by default, as described in [charter signed with Linux Foundation when AsyncAPI joined the foundation](https://github.com/asyncapi/community/blob/master/CHARTER.md). It is not a subject that other TSC members vote on. 
```

* Allowed to commit to regularly
* Review other PR's
* Have voting rights
* Invited to TSC meetings

See current [TSC members](https://www.asyncapi.com/community/tsc)

> ## Usefull links

Reviewer: I suggest we remove this section, these links are already mentioned elsewhere

> Make sure all your PR checks pass(turn to green) to ensure your PR is merged quickly.   

Reviewer: ```suggestion
Make sure all your PR checks pass (turn to green) to ensure your PR is merged quickly.   

```

# BECOME A  TSC MEMBER!

In this article, you will learn how to become a member of the AsyncAPI Technical Steering Committee (TSC). You will know how to contribute, how to become a committer(maintainer), and realize the benefits of being a TSC member.

The following areas are covered:
* What is the TSC?
* Why TSC was formed?
* Benefits of becoming a TSC member
* How to become a TSC member?

If you prefer video, watch our [YouTube video](https://www.youtube.com/watch?v=uG_aLF9Z1F0) on how to become a TSC member.

Definition of common terms in this document:
* **Maintainers:** Are Contributors who have earned the ability to modify (“commit”) source code, documentation, or other technical artifacts in a project’s repository (“Committers”); They are also referred to as CODEOWNERS on [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
* **Contributors:** Include anyone in the community that contributes code, documentation, pull request reviews, or other technical artifacts to the Project (“Contributors”);

## What is the Technical Steering Committee?

> TSC is a group of maintainers and Committers that contribute to the creation and maintenance of AsyncAPI Initiative.

Reviewer: ```suggestion
TSC is a group of maintainers that contribute to the creation and maintenance of AsyncAPI Initiative.
```

since we agreed to just refer in document to `maintainers` only then let's not use `committers` as we agreed it means the same

## Why the TSC was formed

> AsyncAPI joined the Linux Foundation in 2021 to guarantee a system of open governance that promotes ownership and neutrality of all participants in the initiative. 

Reviewer: ```suggestion
The concept of TSC was formed when AsyncAPI joined the Linux Foundation in 2021 to guarantee a system of open governance that promotes ownership and neutrality of all participants in the initiative. 

```

 
Some of the principles of the Linux Foundation partnership include:

* No founding members with special privileges
* Equality among all contributors
* Asynchronous decision making 

> * Every committer is a TSC member

Reviewer: ```suggestion
* Every maintainer is allowed to automatically become a TSC member
```

## Benefits of becoming a TSC member

As a TSC member you will enjoy the following benefits: 
 
* Recognition from AsyncAPI community

> * Exposure for your portfolio to employers and company

Reviewer: ```suggestion
* Exposure of your portfolio to potential employers

```

Make sure all your PR checks pass (turn to green) to ensure your PR is merged quickly.   

**Step 4: Invitation to become a maintainer**

After making several contributions to AsyncAPI, maintainers will recognize you. You will then receive an invitation to join the TSC. Look at [this example from the website repository](https://github.com/asyncapi/website/pull/890).

**Step 5: Become a TSC member**

Once you become a maintainer in one of the repositories, you can apply to join a group of [TSC members](https://www.asyncapi.com/community/tsc).

Open a PR in **community** repository to add your name to [the list of members](https://github.com/asyncapi/community/blob/master/TSC_MEMBERS.json). Specify your name, repositories that you maintain and if you are sponsored by some company. For example [see this PR that introduced one of new members](https://github.com/asyncapi/community/pull/277).

Joining TSC is something that you are allowed to do as a maintainer by default, as described in [charter signed with Linux Foundation when AsyncAPI joined the foundation](https://github.com/asyncapi/community/blob/master/CHARTER.md). It is not a subject that other TSC members vote on. 

> See current [TSC members](https://www.asyncapi.com/community/tsc)

Reviewer: ```suggestion
See current [TSC members](https://www.asyncapi.com/community/tsc).
```
